dua/dua.py

```diff
@@ -76,47 +75,60 @@ async def _dua(self, ctx, subject: str):
         em.set_author(name="Fortress of the Muslim", icon_url=ICON)
         await ctx.send(embed=em)
 
+    async def _dualist(self, ctx, prefix):
+        dua_list_message = [f'**Type {prefix}dua <topic>**. Example: `{prefix}dua breaking fast`\n']
```

Comment:
Abstracted this out to be reusable for @commands.command() and slash command functions